test/jdk/ProblemList.txt

```diff
@@ -787,3 +787,8 @@ java/awt/event/MouseEvent/SpuriousExitEnter/SpuriousExitEnter.java 8254841 macos
 java/awt/Focus/AppletInitialFocusTest/AppletInitialFocusTest1.java 8256289 windows-x64
 java/awt/FullScreen/TranslucentWindow/TranslucentWindow.java 8258103 linux-all
 java/awt/Focus/FrameMinimizeTest/FrameMinimizeTest.java 8016266 linux-x64
+
+############################################################################
+# java/foreign
```

Comment:
The convention used in the comments on each section is to use the group name rather than the package name, in this case, it's jdk_foreign.

Reply:
OK, we'll fix this.